# Goal
atlas_msgs/AtlasSimInterface goal
---
# Result
atlas_msgs/AtlasSimInterfaceState end_state
bool success
---
# Feedback
atlas_msgs/AtlasSimInterfaceState state
